geode-core/src/main/java/org/apache/geode/management/internal/operation/RebalanceOperationPerformer.java

```diff
@@ -35,11 +35,11 @@ public static RebalanceResult perform(Cache cache, RebalanceOperation parameters
       RebalanceCommand rebalanceCommand = new RebalanceCommand();
       rebalanceCommand.setCache(cache);
 
-      ResultModel result = rebalanceCommand
-          .rebalance(arr(parameters.getIncludeRegions()), arr(parameters.getExcludeRegions()), -1,
-              parameters.isSimulate());
+      ResultModel result = rebalanceCommand.rebalanceCallable(arr(parameters.getIncludeRegions()),
+          arr(parameters.getExcludeRegions()), parameters.isSimulate()).call();
 
-      if (result.getStatus().equals(Result.Status.ERROR)) {
+      if (result.getStatus().equals(Result.Status.ERROR)
```

Comment:
If you make the previous change I requested then this code does not need to change.